src/microbe_stage/MicrobeStage.cs

```diff
@@ -893,32 +899,64 @@ private void UpdateLinePlayerPosition()
     {
         if (Player == null || Player?.Dead == true)
         {
-            foreach (var chemoreceptionLine in chemoreceptionLines)
-                chemoreceptionLine.Visible = false;
+            foreach (var chemoreception in chemoreceptionLines)
+                chemoreception.Line.Visible = false;
 
             return;
         }
 
-        var position = Player!.GlobalTransform.origin;
+        var position = Player!.Translation;
 
-        foreach (var chemoreceptionLine in chemoreceptionLines)
+        foreach (var chemoreception in chemoreceptionLines)
         {
-            if (chemoreceptionLine.Visible)
-                chemoreceptionLine.LineStart = position;
+            if (chemoreception.Line.Visible == false)
```

Comment:
Is Visible a nullable type? If not it should not be explicitly compared against a boolean, that's also not the Thrive style.

Reply:
Visible is not nullable here. Which style would you use? I couldn't find a definitive answer in the existing code or style guide.

Reply:
`if (!chemoreception.Line.Visible)`